### DISCUSSION TITLE
Option cellselector to filter slide

### DISCUSSION THREAD

**Discussion by sarussss** [davidjerleke/embla-carousel#435] [Help] [answered]
Demo https://flickity.metafizzy.co/options.html#cellselector
https://codepen.io/jeffglenn/pen/YYwMao
Best way make it with emba carousel? @davidjerleke pls help me.

**Comment by davidjerleke** [comment, marked as answer]
Hi @sarussss,
Thanks for your question. What package are you using? Vanilla? React? Vue?

Best,
David

**Reply by sarussss** [reply]
Hi @davidjerleke , I'm using Vanilla package. 
I'm using start by including the Embla Carousel script from a CDN with a script tag
https://www.embla-carousel.com/get-started/cdn/

**Reply by davidjerleke** [reply]
@sarussss in that case you can do the following:

```html
<script type="text/javascript">
  const emblaNode = document.querySelector('.embla');
  const slideNodes = Array.from(emblaNode.querySelectorAll('.your-slide-nodes-selector'));
  const options = { loop: false };

  const embla = EmblaCarousel({
    root: emblaNode,
    slides: slideNodes
  }, options);
</script>
```

⚠️ Please note:
- Replace `.your-slide-nodes-selector` with the selector you want.
- Any other element inside the slide container that you don't treat as slides either can't have any size (width or height), or has to be position absolute for this to work.

Best,
David

**Reply by sarussss** [reply]
Thank you for the response.

**Reply by davidjerleke** [reply]
Hi @sarussss,

This feature has now been released with [v7.1.0](https://github.com/davidjerleke/embla-carousel/releases/tag/v7.1.0) and is part of the official [options](https://www.embla-carousel.com/api/options/) API.

Best,
David
